iwyu_include_picker.h

```diff
@@ -230,8 +238,15 @@ class IncludePicker {
   IncludeMap filepath_include_map_;
 
   // A map of all quoted-includes to whether they're public or private.
-  // Quoted-includes that are not present in this map are assumed public.
-  map<string, IncludeVisibility> filepath_visibility_map_;
+  // Quoted-includes that are not present in this map or the one below
```

Comment:
This is a little misleading as the path map below does not contain quoted-includes.

How about `Includes that are not present in this map or by path in the one below are assumed public.`

Reply:
Rephrased in a different way.